### PR TITLE
CM-151: included openSearch and openSearch Dashboards setup in dockerized instance

### PR DESCRIPTION
**Pull request by sniedzielski**
TICKET: https://openimis.atlassian.net/browse/CM-151

### CODE REVIEW

File: docker-compose.openSearch.yml [hunk 1]
```diff
@@ -0,0 +1,50 @@
+version: '2.4'
+services:
+  opensearch:
+    build:
+      context: .
+      dockerfile: Dockerfile-openSearch-dev-
+    ports:
```
Comment (dborowiecki) [inline docker-compose.openSearch.yml:7]: I don't think we should expose ports of open search by default as it's a backend for the dashboards. Please let me know if there is anything in this container that user should have access to directly (like UI that is not proxies by front end)

File: docker-compose.openSearch.yml [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+      - "9200:9200"
+      - "9300:9300"
+    env_file:
+      - ".env.openSearch"
+    environment:
+      - "discovery.type=${DISCOVERY_TYPE:-single-node}"
+      - "cluster.name=${CLUSTER_NAME:-my_opensearch_cluster}"
+      - "http.port=${HTTP_PORT:-9200}"
+      - "plugins.security.ssl.http.enabled=${SLL_HTTP_ENABLED:-false}"
+      # Add any other necessary environment variables here
```
Comment (dborowiecki) [inline docker-compose.openSearch.yml:17]: This comment is redundant

**Review by dborowiecki — CHANGES_REQUESTED** [review]
Looks solid, I left 2 comments regarding minor improvements

File: docker-compose.openSearch.yml (hunk 2 above)
Comment (sniedzielski) [reply docker-compose.openSearch.yml]: Ok, I've just removed.

File: docker-compose.openSearch.yml (hunk 1 above)
Comment (sniedzielski) [reply docker-compose.openSearch.yml]: Yes, You are right. After going into 9200 there is web popup to provide auth credentials. After login you should see some json with basic information. No more information are displayed on port 9200. Therefore we can skip exposing those ports. I've just checked this on my local computer and still works after removing this part.
Screenshot from 2023-06-19 11-10-40

**Comment by sniedzielski** [comment]
@dborowiecki You can re-check PR.

**Review by dborowiecki — APPROVED** [review]

**Pull request merged**